iOS_SDK/OneSignalSDK/Source/OSMessagingController.m

```diff
@@ -140,8 +141,20 @@ - (instancetype)init {
     return self;
 }
 
-- (void)didUpdateMessagesForSession:(NSArray<OSInAppMessage *> *)newMessages {
+- (void)updateInAppMessagesFromCache {
+    self.messages = [OneSignalUserDefaults.initStandard getSavedCodeableDataForKey:OS_IAM_MESSAGES_ARRAY
+                                                                      defaultValue:[NSArray new]];
+    [self evaluateMessages];
+    [self deleteOldRedisplayedInAppMessages];
```

Comment:
won't hurt to have this in here, but for consistency, can we keep this only on the updateInAppMessagesFromOnSession method?

Reply:
yes of course